caiman/base/movies.py

```diff
@@ -535,7 +535,7 @@ def return_cropped(self, crop_top=0, crop_bottom=0, crop_left=0, crop_right=0, c
             crop_begin/crop_end: (undocumented)
         """
         t, h, w = self.shape
-        ret = np.resize(self, ( t - crop_end - crop_begin, h - crop_bottom - crop_top, w - crop_right - crop_left))
+        ret = np.zeros(( t - crop_end - crop_begin, h - crop_bottom - crop_top, w - crop_right - crop_left))
         ret[:,:,:] = self[crop_begin:t - crop_end, crop_top:h - crop_bottom, crop_left:w - crop_right]
```

Comment:
Is there a reason to cast the movie object into a ndarray vs. keeping it as a movie since it is not cropping inline (and thus shouldn't have any effects downstream)?

Reply:
It doesn't much matter (and it should be trivial to pull it back into being a movie if desired).